File: TripKitAndroid/src/main/java/com/skedgo/tripkit/RealTimeTripUpdateReceiver.java
package com.skedgo.tripkit;


import io.reactivex.Flowable;
import kotlin.Pair;
import com.skedgo.tripkit.routing.Trip;
import com.skedgo.tripkit.routing.TripGroup;

public interface RealTimeTripUpdateReceiver {
  Flowable<Pair<Trip, TripGroup>> startAsync();
  void stop();
}

File: TripKitAndroid/src/main/java/com/skedgo/tripkit/RealTimeTripUpdateReceiver.kt
package com.skedgo.tripkit;


import com.skedgo.tripkit.routing.Trip;
import com.skedgo.tripkit.routing.TripGroup;

import io.reactivex.Flowable;
import kotlin.Pair;

public interface RealTimeTripUpdateReceiver {
    Flowable<Pair<Trip, TripGroup>> startAsync();

    void stop();
}